Dockerfile.tester

```diff
@@ -49,7 +50,7 @@ FROM base
 ARG DATASET_NAME
 ARG SERVICE_PACKAGE=ibpc_tester
 ARG SERVICE_EXECUTABLE_NAME=ibpc_tester
-ARG SPLIT_TYPE
+ARG SPLIT_TYPE=val
```

Comment:
Unrelated?

Reply:
it's unrelated but as mentioned in the description, without this there will be a runtime error if user does not pass -e SPLIT_TYPE=val to the docker run command since the ros args passed to the context will be `-p split_type:=` which is invalid syntax.